server/android/src/android.rs

```diff
@@ -2,167 +2,89 @@
 #[allow(non_snake_case)]
 pub mod android {
     extern crate jni;
-
-    use std::io::Write;
     use std::path::PathBuf;
+    use std::sync::Mutex;
     use std::thread::{self, JoinHandle};
-    use std::{collections::HashMap, sync::Mutex};
 
-    use jni::sys::{jchar, jshort};
-    use rcgen::generate_simple_self_signed;
+    use jni::sys::jchar;
     use repository::database_settings::DatabaseSettings;
-
-    use server::certs::{PRIVATE_CERT_FILE, PUBLIC_CERT_FILE};
     use server::start_server;
     use service::settings::{ServerSettings, Settings};
     use tokio::sync::mpsc;
 
     use self::jni::objects::{JClass, JString};
-    use self::jni::sys::{jlong, jstring};
     use self::jni::JNIEnv;
 
     use android_logger::Config;
     use log::Level;
 
-    use once_cell::sync::Lazy;
-
-    /// Handle for a running remote_server instance
-    struct RunningServerHandle {
-        /// Channel to signal the remote_server to exit
+    struct ServerBucket {
         off_switch: mpsc::Sender<()>,
-        /// JoinHandle for the running remote_server thread
         thread: JoinHandle<()>,
     }
 
-    /// Manages a set of RunningServerHandle
-    /// Each running server is associated with an integer handle which can be passed to the Java
-    /// world, e.g. the handle can be used to shutdown the server from Java.
-    struct ServerBucket {
-        // next handle ready to be used
-        next_handle: i64,
-        map: HashMap<i64, RunningServerHandle>,
-    }
-
-    impl ServerBucket {
-        /// Returns an integer handle to the running server
-        fn insert(&mut self, server: RunningServerHandle) -> i64 {
-            let handle = self.next_handle;
-            self.next_handle = handle + 1;
-            self.map.insert(handle, server);
-            handle
-        }
-    }
-
-    static SERVER_BUCKET: Lazy<Mutex<ServerBucket>> = Lazy::new(|| {
```

Comment:
@clemens-msupply was there any reason to pass on the handle (which is really an index of a `map` in this case) to  native and then back ? 
Also static Mutex was recently stabilised, so used that (I dont' think it was possible when android.rs was first created)

Reply:
This was for the use-case that you start the server multiple times on different ports and then query or shut them down based on the handle. But I think this doesn't make sense for the remote server. Think it would make more sense to check if the server is already running if you call startServer and not allow to start another server.

Cool that there is static Mutex now!